components/partials/fee-discounts/overview.vue

```diff
@@ -97,36 +92,61 @@ import {
   StatusType
 } from '@injectivelabs/utils'
 import Vue from 'vue'
-import { ZERO_IN_BASE } from '@injectivelabs/sdk-ui-ts'
+import { format, intervalToDuration } from 'date-fns'
 import {
   cosmosSdkDecToBigNumber,
-  FeeDiscountAccountInfo
+  FeeDiscountAccountInfo,
+  FeeDiscountSchedule
 } from '@injectivelabs/sdk-ts'
+import { ZERO_IN_BASE } from '@injectivelabs/sdk-ui-ts'
 import { UI_DEFAULT_MIN_DISPLAY_DECIMALS } from '~/app/utils/constants'
-import VItem from '~/components/partials/common/stats/item.vue'
+import { getAbbreviatedVolume } from '~/app/utils/market'
 
 export default Vue.extend({
-  components: {
-    VItem
-  },
-
   data() {
     return {
       status: new Status(StatusType.Loading),
-
+      slideIndex: 0,
       UI_DEFAULT_MIN_DISPLAY_DECIMALS
     }
   },
 
   computed: {
+    apr(): BigNumberInBase {
+      return this.$accessor.params.apr
+    },
+
+    aprToFormat(): string {
+      const { apr } = this
+
+      return apr.times(100).toFormat(2)
+    },
+
     isUserWalletConnected(): boolean {
       return this.$accessor.wallet.isUserWalletConnected
     },
 
+    lastUpdateTimestamp(): string {
+      const { feeDiscountAccountInfo } = this
+
+      if (!feeDiscountAccountInfo || !feeDiscountAccountInfo.accountTtl) {
+        return '-'
+      }
+
+      return format(
+        Number(feeDiscountAccountInfo.accountTtl.ttlTimestamp) * 1000,
+        'yyyy-MM-dd HH:mm:ss (zzz)'
+      )
+    },
+
     feeDiscountAccountInfo(): FeeDiscountAccountInfo | undefined {
```

Comment:
Lets move store variables computed properties to the top, same with `feeDiscountSchedule`